멘토링/0915.md

```diff
@@ -0,0 +1,38 @@
+## 팀 소개
```

Comment:
Doc in Korean is bad to oversea engineers.
This documentation shouldn't be involoved in this patch.